src/routes/classes/+page.svelte

```
@@ -604,7 +605,8 @@
 							type="button"
 							class="btn btn-ghost btn-sm rounded-full text-base-content/60"
 							onclick={async () => {
-								const href = featureAnnouncement.ctaHref || '/classes';
+								const resolveHref = resolve as unknown as (href: string) => string;
+								const href = resolveHref(featureAnnouncement.ctaHref ?? '/classes');
 								await dismissFeatureSpotlight();
 								await goto(href);
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)